operator/e2e/dependencies.yaml

```diff
@@ -48,3 +48,11 @@ helmCharts:
     version: v25.3.4
     namespace: gpu-operator
     repoURL: https://helm.ngc.nvidia.com/nvidia
+
```

Comment:
Can you Include the cert manager images so that it's faster if you have them in docker locally?

Reply:
@gflarity , definitely it will be faster, however it will make the repo larger. Would you still like me to include this images ?

Reply:
Yes, definitely